src/components/input/input.css

```diff
@@ -23,3 +28,20 @@
   font-style: italic;
   color: var(--palette-neutral-mid-dark);
 }
+
+.invalid-icon {
+  box-sizing: border-box;
+  height: 100%;
+  pointer-events: none;
+  position: absolute;
+  right: 0;
+}
+
+.invalid {
```

Comment:
I just realized. I might not have to rely on a property for an invalid state. 

TODO: Try out psuedo state for invalid. A required input that is invalid will somehow style itself differently -- so can I utilize whatever is happening under the hood? Are there any cons to using this technique?